cmd/talosctl/cmd/mgmt/cluster/create/create_dev.go

```diff
@@ -781,7 +782,7 @@ func mergeKubeconfig(ctx context.Context, clusterAccess *access.Adapter) error {
 
 	_, err = os.Stat(kubeconfigPath)
 	if err != nil {
-		if !os.IsNotExist(err) {
+		if !errors.Is(err, fs.ErrNotExist) {
```

Comment:
why change, is it deprecated? It's easy for new code to miss using `io/fs` and just continue using `os.IsNotExist`

Reply:
🆒 👍